Base/res/apps/BrowserSettings.af

```diff
@@ -3,3 +3,4 @@ Name=Browser Settings
 Executable=/bin/BrowserSettings
 Category=Settings
 Description=Configure Browser
+ExcludeFromTaskbar=true
```

Comment:
"exclude from taskbar" confused me, since the applications could very well show up on the taskbar. Maybe rename to `ExcludeFromMainMenu` or something similar?

Reply:
:+1: Good point, looks like we call it "system menu" elsewhere so I'll go with that